### ISSUE TITLE
Crashing server Crate and pipes

### ISSUE THREAD

**Issue by SirDizarm** [MysticMods/Druidcraft#106]
**Describe the bug:**
Playing Valhelsia 3 and Basic Logistical Transporter(item pipes) if they are connected to a "crate (chest from druidcraft)" and break, or pick the crate up. It breaks the server

**Comment by noobanidus** [comment]
Unfortunately we do need a crash report for this one.

**Comment by SirDizarm** [comment]
[
[crash-2021-03-24_11.43.59-server.txt](https://github.com/MysticMods/Druidcraft/files/6196406/crash-2021-03-24_11.43.59-server.txt)
](url)

**Comment by SirDizarm** [comment]
Can easy replecate if having Mekanism logisticpipe connected in one end where it pulls, and the other where it push, and then brake any of the crates or move them some whay.

**Comment by noobanidus** [comment]
Okay, I think this has potentially been fixed, just not released yet.

**Comment by SirDizarm** [comment]
> Okay, I think this has potentially been fixed, just not released yet.

Ok, well I think it is a huge issue that might need a small update just to get it out on servers, all Valhelsia 3 servers have this issue and there are a lot of youtubers playing, if its possible to make it work that is.

**Comment by noobanidus** [comment]
> Ok, well I think it is a huge issue that might need a small update just to get it out on servers, all Valhelsia 3 servers have this issue and there are a lot of youtubers playing, if its possible to make it work that is.

I agree, I'm going to pester Vulpie about it after our D&D game today.

**Comment by fullshadow** [comment]
Today our server also encountered this problem. I would like to know if there is any progress in the solution?
[crash-2021-05-01_19.00.56-server.txt](https://github.com/MysticMods/Druidcraft/files/6409917/crash-2021-05-01_19.00.56-server.txt)

**Comment by Gr3atsaga** [comment]
I have also been afflicted by this issue.  I was able to circumvent the issue by changing my location in my player .dat file to an area that would not load the two... but when I reloaded the chunk in which I had a logistic pipe connected to a druidcraft crate, the server crashed again.

**Comment by VallenFrostweaver** [comment]
Still happening. Here's another crash report: 
[crash-2021-09-09_17.11.24-server.txt](https://github.com/MysticMods/Druidcraft/files/7139832/crash-2021-09-09_17.11.24-server.txt)

**Comment by noobanidus** [comment]
Please all try the version ending in .43, I think this solve most of the issues.

**Comment by Gr3atsaga** [comment]
I think you meant the .53 version.  I tested that version and it seems like the issue with the logistical transporter is fixed.  I broke both the transporter and the DruidCraft crate, picked them both up, let items pipe into the crate etc. and no crash.  This specific issue seems to be resolved.
